applications/serial_lte_modem/Kconfig

```diff
@@ -82,6 +81,17 @@ config SLM_INTERFACE_PIN
 	default 6 if SLM_CONNECT_UART_0
 	default 31 if SLM_CONNECT_UART_2
 
+#
+# Socket
+#
+config SLM_SOCKET_RX_MAX
+	int "Maximum RX buffer size for socket data receive"
```

Comment:
```suggestion
	int "Maximum RX buffer size for receiving socket data"
```

Reply:
Resolved but not fixed?

Reply:
Sorry, update lost.
will add in next PR#3616, more KConfig changes